src/dash-table/components/Export/index.tsx

```diff
@@ -0,0 +1,40 @@
+import XLSX from 'xlsx';
+import React from 'react';
+import { IDerivedData, IVisibleColumn } from 'dash-table/components/Table/props';
+import { createWorkbook, createHeadings, createWorksheet } from './utils';
+import getHeaderRows from 'dash-table/derived/header/headerRows';
+
+interface IExportButtonProps {
+    export_format: string;
+    virtual_data: IDerivedData;
+    columns: IVisibleColumn[];
+    export_headers: string;
+}
+
+export default React.memo((props: IExportButtonProps) => {
```

Comment:
This is 👍. Just want to let you know that you can also declare it with generics like so:
```
export default React.memo<IExportButtonProps>(props => {
```